### PR TITLE
Sync Dutch translation file to English + corrections

### PR DESCRIPTION
**Pull request by h3ll0fr13nd**
Mostly rearranged the lines to get equal to the English translation file for easier future maintenance. Also did some small spelling corrections and improved a few translations when I matched the Dutch and English files.

### CODE REVIEW

**Comment by frozeman** [comment]
@h3ll0fr13nd you json is faulty, please check if you have all commas correct on the line endings and the last line don't have any.
`/home/travis/build/ethereum/mist/interface/i18n/mist.nl.i18n.json: Can't load`

File: interface/i18n/mist.nl.i18n.json [hunk 1]
```diff
@@ -113,21 +135,21 @@
         "startScreen": {
             "runningNodeFound": "Lopende Ethereum node gevonden!",
             "startingNode": "Ethereum node starten...",
+            "stoppingNode": "Ethereum node afsluiten...",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: Suggestion: `Ethereum node stoppen...`.

File: interface/i18n/mist.nl.i18n.json [hunk 2]
```diff
@@ -113,21 +135,21 @@
...
-            "stoppingNode": "Ethereum node afsluiten...",
+            "nodeBinaryNotFound": "Geen Ethereum node uitvoerbaar bestand gevonden!<br><small> <a href='https://www.ethereum.org/cli#geth' class='button' target='_blank'> Start er a.u.b. handmatig een. </a> </small>",
             "nodeStarting": "Ethereum node opstarten...",
             "nodeStarted": "Ethereum node gestart",
             "nodeConnected": "Ethereum node verbonden",
             "nodeStopping": "Ethereum node afsluiten...",
             "nodeStopped": "Ethereum node gestopt",
             "nodeError": "Ethereum node verbindingsfout :'(",
             "unableToBindPort": "Ethereum node kan niet opstarten. Draait er al een?",
+            "nodeSyncing": "Ethereum node moet synchroniseren, even gedult alstublieft...",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: I believe its `geduld` instead of `gedult`.

File: interface/i18n/mist.nl.i18n.json [hunk 3]
```diff
@@ -89,15 +111,15 @@
...
             "blockNumber": "Jouw laatste bloknummer",
             "timeSinceBlock": "Verstreken tijd sinds laatste blok",
-            "testnetExplain": "Je bent actief op een testnet, VERSTUUR GEEN echte Ether naar deze adressen",
-            "peers": "peers",
             "checkingWhichNetwork": "Netwerk aan het controleren...",
             "mainNetwork": "Main-net",
             "testNetwork": "Test-net",
             "privateNetwork": "Private-net",
             "mainnetExplain": "Je zit op het main Ethereum wereldwijde netwerk",
+            "testnetExplain": "Je bent actief op een testnet, VERSTUUR GEEN echte Ether naar deze adressen",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: The English transaction uses the `__name__` placeholder.
Would be nice if you follow the same pattern.

File: interface/i18n/mist.nl.i18n.json [hunk 4]
```diff
@@ -142,6 +164,18 @@
             }
         },
         "popupWindows": {
+            "updateAvailable": {
+                "newVersionAvailable": "Nieuwe __name__ versie beschikbaar",
+                "version": "Versie",
+                "downloadURL": "Download URL",
+                "checksum": "Checksum",
+                "downloadAndRestart": "Update en herstart",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: `Updaten en opnieuw opstarten` is a bit more common.

File: interface/i18n/mist.nl.i18n.json [hunk 5]
```diff
@@ -142,6 +164,18 @@
...
+                "download": "Download nieuwe versie",
+                "skipUpdate": "Sla update over",
+                "notNow": "Vraag me later opnieuw",
+                "checking": "Controleren op updates voor __name__...",
+                "noUpdateFound": "Geen updates gevonden. Je hebt reeds de laatste versie __name__."
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: Please drop `reeds`.

**Comment by frozeman** [comment]
@h3ll0fr13nd can you please fix @bas-vk suggestions? Thanks @bas-vk

File: interface/i18n/mist.nl.i18n.json [hunk 6]
```diff
@@ -171,12 +205,15 @@
                 "enterPassword": "Voer wachtwoord in om de transactie te bevestigen",
                 "unlocking": "Bevestigen...",
                 "createContract": "Contract aanmaken",
-                "estimatedFee": "Ingeschatte kosten",
-                "estimatedGasError": "Data kon niet worden uitgevoerd, dus het zal alle mogelijke gas gebruiken.",
+                "estimatedFee": "Ingeschatte transactie kosten",
+                "estimatedGasError": "Het lijkt erop dat deze transactie zal mislukken. Als het het toch verzend zou het alle verzonden gas kunnen verbruiken.",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: Something like `De kans is groot dat de transactie zal mislukken.`.
The last sentence isn't correct Dutch.

File: interface/i18n/mist.nl.i18n.json [hunk 7]
```diff
@@ -218,8 +252,12 @@
                 "you": "Je",
                 "etherbase": "Centrale account (Etherbase)",
                 "depositBitcoin": "Bitcoin Storten",
+                "viaShapeshift": "Direct omwisselen via <a href='https://shapeshift.io/' target='_blank'>Shapeshift</a>",
                 "learnIt": "Leer terwijl je wacht",
                 "downloadingBlocks": "Blokken downloaden",
+                "syncMessage": "Blok __displayBlock__ van __highestBlockString__",
+                "syncMessageWithStates": "Blok __displayBlock__ van __highestBlockString__ (Chain structuur __statesPercent__%)",
+                "startingSync": "Gereed maken voor sync..",
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: Probably something like `Wachten op synchronisatie...` is better.
But it depends on the context.

File: interface/i18n/mist.nl.i18n.json [hunk 8]
```diff
@@ -233,23 +271,8 @@
...
-                "version": "Versie",
-                "downloadURL": "Download URL",
-                "checksum": "Checksum",
-                "downloadAndRestart": "Update en herstart",
-                "download": "Download nieuwe versie",
-                "skipUpdate": "Sla update over",
-                "notNow": "Vraag me later opnieuw",
-                "checking": "Controleren op updates voor __name__...",
-                "noUpdateFound": "Geen updates gevonden. Je hebt reeds de laatste versie __name__."
+                    "importFailed": "Kon bestand niet importeren en kreeg: __error__"
```
Comment (bas-vk) [inline interface/i18n/mist.nl.i18n.json]: `Bestand importeren mislukt: __error__`

**Comment by bas-vk** [comment]
LGTM, thanks!

**Pull request merged**